# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: dom/base/Navigator.cpp
# Commit: 08eb9cf235bb
# Full Hash: 08eb9cf235bb5334ef01075d6148e1ea235adf56
# Author: Benjamin VanderSloot <bvandersloot@mozilla.com>
# Date: 2025-06-05 04:07:37
# Regressor Bug: 1969017
# File Overlap Count: 2
# Description:
#   Bug 1969017 - Refactor FedCM to use a managed actor rather than tagging onto WindowGlobalParent - r=anti-tracking-reviewers,emz
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D251617
# ==============================================================================

diff -r ff3135366145 -r 08eb9cf235bb dom/base/Navigator.cpp
--- a/dom/base/Navigator.cpp	Wed Jun 04 19:25:54 2025 +0000
+++ b/dom/base/Navigator.cpp	Wed Jun 04 19:25:54 2025 +0000
@@ -2269,7 +2269,7 @@
 
 NavigatorLogin* Navigator::Login() {
   if (!mLogin) {
-    mLogin = new NavigatorLogin(GetWindow()->AsGlobal());
+    mLogin = new NavigatorLogin(GetWindow());
   }
   return mLogin;
 }